packages/react-ui-core/src/Form/RangeSlider.js

```diff
@@ -3,6 +3,7 @@ import PropTypes from 'prop-types'
 import InputRange from 'react-input-range'
 import classnames from 'classnames'
 import omit from 'lodash/omit'
+import isEqual from 'lodash/isEqual'
 import themed from 'react-themed'
 
 @themed('*', {
```

Comment:
I'm confused as to why we're doing `'*'` for the 1st argument for `themed` instead of `'/^RangeSlider/'`
Wouldn't we be able to avoid doing 
```
        classnames(
          theme.RangeSlider,
          className,
        )}
```
?
Not meant as a criticism, I'm just confused

Reply:
I think I added it here like this because of the InputRange theme and it's weirdness with css modules.  None of them begin with `RangeSlider` unless specified in another file.

Reply:
oh ok. 👍